policy_notification/models.py

```diff
@@ -3,9 +3,17 @@
 from core.utils import get_first_or_default_language
 from insuree.models import Family
 from policy.models import Policy
+from enum import IntEnum
 
 
 class FamilySMS(core_models.BaseVersionedModel):
```

Comment:
Can we change FamilySMS to FamilyNotification?

Reply:
Changed